### ISSUE TITLE
'High Contrast Black' theme Table same text and background color 

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1017]
**Describe the bug**
The Table component has the same color for text and background - white. So table contents are not readable.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/table.html
2. Check the Table component

**Expected behavior**
The table component should have a black background color for HCB theme.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81758931-12c35900-9478-11ea-91c4-a1442b3e0962.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by jacobdevera** [comment]
fixed by #1042
